test262/per-file/index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LibJS test262 per-file results</title>
+    <link rel="shortcut icon" type="image/png" href="/favicon.png" />
+    <link rel="stylesheet" href="../main.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@700&family=Roboto:wght@400;700&display=swap"
+    />
+    <style></style>
+  </head>
+
+  <body>
+    <main>
+      <h1>LibJS test262 per-file result viewer</h1>
+      <p>
+        These are the per-file results of the SerenityOS JavaScript engine,
+        LibJS, running the official ECMAScript conformance test suite test262.
+        All tests are re-run and the results updated automatically for every
+        push to the master branch of the repository on GitHub.
+      </p>
+      <p>
+        The source code of the LibJS test262 runner and aggregation script is
+        available
+        <a
+          href="https://github.com/linusg/libjs-test262"
+          target="_blank"
+          rel="noopener noreferrer"
+          >here</a
+        >, the source code of this website
+        <a
+          href="https://github.com/linusg/libjs-website"
+          target="_blank"
+          rel="noopener noreferrer"
+          >here</a
+        >, and the raw data
+        <a
+          href="https://github.com/linusg/libjs-website/blob/main/test262/data/per-file-master.json"
+          target="_blank"
+          rel="noopener noreferrer"
+          >here</a
+        >
+        for the AST interpreter, and
+        <a
+          href="https://github.com/linusg/libjs-website/blob/main/test262/data/per-file-master-bytecode.json"
+          target="_blank"
+          rel="noopener noreferrer"
+          >here</a
+        >
+        for the bytecode interpreter. (JSON).
+      </p>
+      <p>
+        If you have any questions or want to help out with improving these test
+        scores, feel free to get in touch on the
+        <a
+          href="https://discord.gg/serenityos"
+          target="_blank"
+          rel="noopener noreferrer"
+          >SerenityOS Discord server</a
+        >
+        (<code>#js</code>).
+      </p>
+      <div>
+        <h2>Results</h2>
+        <div class="mode-summary-container">
+          <span id="summary">Loading...</span>
+          <div class="tree-node-status-container">
+            <span id="summary-status" class="hidden tree-node-status"></span>
+          </div>
+        </div>
+        <div style="height: 60px"></div>
```

Comment:
eww, inline CSS! jk, but maybe make this part of a class if we can?

Reply:
oh sure, I guess I just missed it when I was moving these into classes after testing them out